build-clean.xml

```diff
@@ -182,7 +182,7 @@ maintainers may prefer to use this instead of build.xml.
 	</target>
 
 	<target name="libdep-bc" depends="env" unless="lib.bouncycastle.present">
-		<fail message="Bouncycastle not available (or not recent enough); try https://www.bouncycastle.org/download/${bc.jar}"/>
+	  <fail message="Bouncycastle not available (or not recent enough); try https://www.bouncycastle.org/download/${bc.jar} and put it in the ./lib/ dir"/>
```

Comment:
Don’t we have a variable for the directory in which this file is expected? If so, we should use that here.